test/flux_calculator_tests.jl

```diff
@@ -0,0 +1,39 @@
+
+using ClimaCore: Meshes, Domains, Topologies, Spaces, Fields, InputOutput
+using ClimaCoupler: Utilities, Regridder, TestHelper
+using Test
+import ClimaCoupler.FluxCalculator:
+    compute_turbulent_fluxes!, compute_combined_turbulent_fluxes!, CombinedAtmosGrid, PartitionedComponentModelGrid
+import ClimaCoupler.Interfacer: AtmosModelSimulation
+FT = Float64
+
+# test for a simple generic atmos model
+struct DummySimulation{S, C} <: AtmosModelSimulation
+    state::S
+    cache::C
+end
+function compute_turbulent_fluxes!(sim::DummySimulation, csf)
+    sim.cache.flux .= (csf.T_sfc .- sim.state.T) .* sim.cache.κ ./ sim.cache.dz
+end
+struct DummySimulation2{C} <: AtmosModelSimulation
+    cache::C
+end
+
+@testset "compute_combined_turbulent_fluxes!" begin
+    boundary_space = TestHelper.create_space(FT)
+    coupler_fields = (; T_sfc = 310 .* ones(boundary_space))
+    sim =
+        DummySimulation((; T = 300 .* ones(boundary_space)), (; κ = FT(0.01), dz = FT(1), flux = zeros(boundary_space)))
+    model_sims = (; atmos_sim = sim)
+    flux_types = (CombinedAtmosGrid(), PartitionedComponentModelGrid())
+    results = [FT(0.1), FT(0.0)]
```

Comment:
Where do these result values come from?

Reply:
It's from the flux calculation above. I added an explanation. Thanks!